.env.example

```diff
@@ -0,0 +1,11 @@
+SECRET_KEY=your-secret-key
+RENUE_API_KEY=your-denue-api=key
+DATABASE_ENGINE=django.contrib.gis.db.backends.postgis
+DATABASE_HOST=localhost
+DATABASE_NAME=postgres
+DATABASE_USERNAME=postgres
+DATABASE_PASSWORD=your-db-password
+DATABASE_PORT=5432
+DJANGO_DEBUG=True
+GDAL_LIBRARY_PATH=your-gdal-path
+GEOS_LIBRARY_PATH=your-geos-path
```

Comment:
Is this `.env` file available in any place? I tried to run the tests and env variables are needed

Reply:
you should create it on the root directory of project

Reply:
Thanks for the `.env` file, I already used it. 
When I run `python manage.py test` the following exception is raised 
```
raise ImproperlyConfigured("Error loading psycopg2 module: %s" % e)
django.core.exceptions.ImproperlyConfigured: Error loading psycopg2 module: No module named 'psycopg2'
```
I tried to install `psycopg2-binary` but other error raised `error: command 'gcc' failed with exit status 1`

Reply:
hi @EmmaMireyaSm  

change the way you work so as not to install dependencies on your computers, now I use Docker and add the commands to the README

thanks

Reply:
@jmercadoc to run what you want in this step for example :
```
docker-compose exec denue-api python denue/manage.py test api.tests.tests_model_comercial_activity

```

You don't really need to have the app running, right? Testing while modifying code can be easier if you use the code instead of a previously built image. What do you think?